source/fundamentals/crud/read-operations/change-streams.txt

```diff
@@ -142,22 +144,23 @@ Include Pre-images and Post-images
 
 You can configure the change event to contain or omit the following data:
 
-- The **pre-image** which is a document that represents the version of the
-  document before the operation if it exists
-- The **post-image** which is a document that represents the version of the
-  document after the operation if it exists
+- The **pre-image**, which is a document that represents the version of the
+  document before the operation, if it exists
+- The **post-image**, which is a document that represents the version of the
+  document after the operation, if it exists
 
 To receive change stream events that include a pre-image or post-image, you
-must connect to a MongoDB v6.0 or later deployment and set up the following:
+must connect to a deployment running MongoDB v6.0 or later and perform the
+following actions:
```

Comment:
S: This sentence is a little hard to parse. Could we move the version info to an admonition or something?

```suggestion
must perform perform the following actions on your MongoDB deployment:
```